src/styles/elements/_field.scss

```diff
@@ -2,87 +2,54 @@
 $name: '' !default;
 
 .#{$name}__Field {
-  display: grid;
-  gap: 0.25em;
+  @include mercury.Input;
 
   &__Label {
-    color: mercury.$color-grayDark;
-    font-weight: 500;
-    font-size: 0.8em;
-    text-transform: uppercase;
+    @include mercury.Typography--SmallScreenLabel;
+
+    @media (min-width: 600px) {
+      @include mercury.Typography--Label;
+    }
   }
 
   &__InlineError {
+    @include mercury.Typography--SmallScreenCaption;
+
     color: mercury.$color-red;
-    font-size: 0.8em;
+
+    @media (min-width: 600px) {
+      @include mercury.Typography--Caption;
+    }
   }
 }
 
 .#{$name}__NumberInput {
+  @include mercury.Input;
+
   display: grid;
   grid-gap: 0.5em;
   grid-template-columns: repeat(3, auto);
   justify-content: start;
   font-size: 0.8em;
 
   input {
-    width: 3.5em;
-    padding: 0.5em 0.75em;
-    line-height: 1em;
-    border: 1px solid mercury.$color-grayLight;
-    border-radius: 0.2em;
-
-    &:focus {
-      border-color: mercury.$color-blue;
-    }
-
-    &:invalid {
-      border-color: mercury.$color-red;
-    }
+    height: 2.25em;
+    font-size: 0.8em;
```

Comment:
@dangodev can we use `em`s instead of `rem`s for things we want to resize? that way we can just set the font size and the input will adjust accordingly

Reply:
The reason for `rem`s is the “numeric“ input here: https://mercury.manifold.now.sh/?path=/story/input--input

It uses `14px IBM Plex Sans` rather than `16px -system`. So `em`s make that a different height.

Maybe that’s OK? I just figured it might be weird side-by-side.

Also I found that without setting an explicit height, and using padding, it cut off descenders like `g`, `j`, etc. Anyway, those are all the considerations—I’m fine with `em`s as anything can be overridden.